### PR TITLE
chore: update rollouts manager to 0.0.2 with crds

### PR DESCRIPTION
**Pull request by ishitasequeira**
**What type of PR is this?**
> /kind bug
> /kind cleanup
> /kind failing-test
> /kind enhancement
> /kind documentation
> /kind code-refactoring

**What does this PR do / why we need it**:
Upgrade rollouts-manager version and crds

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: test/openshift/e2e/parallel/1-090_validate_permissions/01-assert.yaml [hunk 1]
```diff
@@ -1,9 +1,309 @@
...
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: openshift-gitops-operator
+    operators.openshift.io/infrastructure-features: '["disconnected"]'
+    operators.operatorframework.io/builder: operator-sdk-v1.10.0+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/redhat-developer/gitops-operator
+    support: Red Hat
   name: gitops-operator.v1.8.0
-  namespace: openshift-gitops-operator
+  namespace: placeholder
```
Comment (varshab1210) [inline test/openshift/e2e/parallel/1-090_validate_permissions/01-assert.yaml:177]:
```suggestion
  namespace: openshift-gitops-operator
```

**Comment by varshab1210** [comment]
Platform issue

/retest

**Comment by varshab1210** [comment]
Platform issue

/retest

**Review by reginapizza — APPROVED** [review]

**Pull request merged**